Tests/Marketplace/upload_packs.py

```diff
@@ -570,8 +565,65 @@ def add_private_packs_to_index(index_folder_path: str, private_index_path: str):
             update_index_folder(index_folder_path, d.name, d.path)
 
 
+def was_private_pack_updated(private_index_json, public_index_json):
+    """
+    Checks if Content commit hash of each private pack in private and public index files are equal. If there is at
+        least one private pack that has a different content commit hash, it tells us that this pack was updated and
+        should be uploaded. So, an upload flow should NOT be skipped.
+
+    Args:
+        private_index_json (dict) : The private index file.
+        public_index_json (dict) : The public index file.
+
+    Returns:
+        (bool) True if Content commit hash of at least one private pack was changed.
+
+    """
+    id_to_commit_hash_from_public_index = {}
+    for private_pack in public_index_json.get("packs"):
+        pack_id = private_pack.get("id")
+        content_commit_hash = private_pack.get("contentCommitHash", "")
+        id_to_commit_hash_from_public_index[pack_id] = content_commit_hash
+
+    for private_pack in private_index_json.get("packs"):
+        pack_id = private_pack.get("id")
+        content_commit_hash = private_pack.get("contentCommitHash", "")
+        if id_to_commit_hash_from_public_index[pack_id] != content_commit_hash:
```

Comment:
Please handle the case that the `pack_id` from the public index is not in the private index (e.g. a pack was deleted, not supposed to happen but better to be on the safe side)

Reply:
ignoring this one as we discussed.